mslice/scripting/helperfunctions.py

```diff
@@ -157,7 +147,7 @@ def add_cut_lines(script_lines, plot_handler):
 
 def add_cut_lines_with_width(errorbars, script_lines, cuts, plot_handler):
     """Adds the cut statements for each interval of the cuts that were plotted"""
-    more_than_one_ws = len(plot_handler._cut_plotter_presenter._cut_cache) > 1
+    #more_than_one_ws = len(plot_handler._cut_plotter_presenter._cut_cache) > 1
```

Comment:
From the removal of the `if` below I assume this is no longer needed. Can it be removed?